src/megameklab/com/ui/aerospace/AdvancedAeroStructureTab.java

```diff
@@ -517,35 +517,47 @@ public void armorPointsChanged(int location, int front, int rear) {
 
     @Override
     public void autoAllocateArmor() {
-        for (int loc = 0; loc < getJumpship().locations(); loc++) {
+        for (int loc = 0; loc < getJumpship().locations() - 1; loc++) {
             getJumpship().initializeArmor(0, loc);
         }
         
-        // divide armor among positions, with more toward the front
+        // divide armor (in excess of bonus from SI) among positions, with more toward the front
+        int bonusPerFacing = (int) Math.floor(UnitUtil.getSIBonusArmorPoints(getJumpship()) / 6.0);
```

Comment:
Is that 6 a constant/value we could get from somewhere instead of hardcode it here?